eth/backend.go

```diff
@@ -209,11 +193,9 @@ func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
 	if err != nil {
 		return nil, err
 	}
-	eth.engine.VerifyHeader(eth.blockchain, eth.blockchain.CurrentHeader(), true) // TODO think on it
 
-	// BOR changes
-	eth.APIBackend.gpo.ProcessCache()
-	// BOR changes
+	// NOT IN GETH
+	eth.engine.VerifyHeader(eth.blockchain, eth.blockchain.CurrentHeader(), true) // TODO think on it
```

Comment:
@ferranbt Can you check if we need this anymore?

Reply:
Not sure what this is doing here, could it be used to preload the snapshots in Bor?

Reply:
Can you remove this line and uncomment https://github.com/maticnetwork/bor/blob/krishna/updated-contract-call-flow/core/blockchain.go#L350-L353 ? I have commented it and added this line hear since `eth.engine` was null before.

Reply:
Got it